service/src/main/java/uk/nhs/adaptors/gp2gp/transformJsonToXmlTool/XmlSchemaValidator.java

```diff
@@ -0,0 +1,100 @@
+package uk.nhs.adaptors.gp2gp.transformjsontoxmltool;
```

Comment:
I think "uk.nhs.adaptors.gp2gp.transformJsonToXmlTool" will be better for package name as IDE checkstyle tool shouts about it

Reply:
fixed this.